README.md

```diff
@@ -31,7 +32,7 @@ import dotenv from "dotenv";
 
 dotenv.config();
 
-const dbUrl = process.env === 'test' ? process.env.TEST_DB : process.env.DATABASE_URL;
+const dbUrl = process.env.NODE_ENV === 'test' ? process.env.TEST_DB : process.env.DATABASE_URL;
```

Comment:
This is good, maybe in the future, we might add some `configuration` files. 
This will help determine the `production` environment also. 
E.g, we might have `test`, `development`, `staging` and `production`.